mix.lock

```diff
@@ -1,24 +1,24 @@
-%{"certifi": {:hex, :certifi, "0.3.0"},
-  "combine": {:hex, :combine, "0.7.0"},
-  "connection": {:hex, :connection, "1.0.2"},
+%{"certifi": {:hex, :certifi, "0.3.0", "389d4b126a47895fe96d65fcf8681f4d09eca1153dc2243ed6babad0aac1e763", [:rebar3], []},
```

Comment:
I would leave .gitignore and just reset this particular file, as there's no need to update it unless mix.exs was also changed.